app/actions/knots.js

```diff
@@ -265,23 +268,35 @@ export function downloadKnot(knot: string) {
       .catch();
 }
 
-export function loadValues(knot: string, uuid: string) {
+export function loadValues(knot: string, uuid: string, seedingState?: boolean) {
   return (dispatch: (action: actionType) => void) => {
     dispatch({
       type: LOADING_KNOT
     });
     return axios
       .post(`${baseUrl}/knots/load`, { knot, uuid })
       .then((response) => {
-        dispatch({
-          type: LOADED_KNOT,
-          tap: response.data.tap,
-          target: response.data.target,
-          tapConfig: response.data.tapConfig,
-          targetConfig: response.data.targetConfig,
-          knotName: response.data.name,
-          schema: response.data.schema
-        });
+        if (seedingState) {
+          dispatch({
+            type: SEEDING_STATE,
+            tap: response.data.tap,
+            target: response.data.target,
+            tapConfig: response.data.tapConfig,
+            targetConfig: response.data.targetConfig,
+            knotName: response.data.name,
+            schema: response.data.schema
+          });
+        } else {
+          dispatch({
+            type: LOADED_KNOT,
+            tap: response.data.tap,
+            target: response.data.target,
+            tapConfig: response.data.tapConfig,
+            targetConfig: response.data.targetConfig,
+            knotName: response.data.name,
+            schema: response.data.schema
+          });
+        }
```

Comment:
Instead of the `if else` block

```
          dispatch({
            type: seedingState ? LOADED_KNOT : SEEDING_STATE,
            tap: response.data.tap,
            target: response.data.target,
            tapConfig: response.data.tapConfig,
            targetConfig: response.data.targetConfig,
            knotName: response.data.name,
            schema: response.data.schema
          });
```